### ISSUE TITLE
Extra GTM <noscript> added when navigating between pages

### ISSUE THREAD

**Issue by jyclam** [FightPandemics/FightPandemics#1190]
**Steps to recreate:**
1. Sign in to your account 
2. Go to homepage on staging https://staging.fightpandemics.work/
3. Click on **any** links eg `About Us`, `Help Board`, `Request Help`, `Offer Help`...
4. Observe that an extra <noscript> tag is added 

It's a GTM related element
```
      <iframe src="https://www.googletagmanager.com/ns.html?id=undefined&gtm_auth=&gtm_preview=&gtm_cookies_win=x"
        height="0" width="0" style="display:none;visibility:hidden" id="tag-manager"></iframe>
```

![Screen Shot 2020-07-16 at 12 52 21 PM](https://user-images.githubusercontent.com/26049767/87716488-c50ee980-c763-11ea-9922-2e13f26dd06f.png)

Looks like every time we call `TagManager.initialize` it inserts a new one, https://github.com/alinemorelli/react-gtm/blob/master/src/TagManager.js#L43

**Comment by nr431207** [comment]
@idjevm I think this is potentially due to the `TagManager.initialize` with UserIs in the navBar?

**Comment by idjevm** [comment]
@nr431207 Yes def, I believe it is because of that. What if we move the initialize to when the user signs in? So we add the userId to the datalayer just then (not sure if it will be kept on the datalayer, but we can test).

**Comment by nr431207** [comment]
@idjevm yes will try that, it shouldn't affect it being kept on the datalayer.

**Comment by idjevm** [comment]
Resolved by #1212
